includes/view/AngelTypes_view.php

```diff
@@ -322,8 +333,62 @@ function AngelType_view_members(AngelType $angeltype, $members, $admin_user_ange
             $ifsg_confirmed = $member->license->ifsg_confirmed;
             $member['ifsg_certificate'] = certificateIcon($ifsg_confirmed, $member->license->ifsg_certificate);
             if (config('ifsg_light_enabled')) {
-                $member['ifsg_certificate_light'] = certificateIcon($ifsg_confirmed, $member->license->ifsg_certificate_light);
+                $member['ifsg_certificate_light'] = certificateIcon(
+                    $ifsg_confirmed,
+                    $member->license->ifsg_certificate_light
+                );
+            }
+        }
+        $goodie_actions = [];
+        if (
+            auth()->can('angeltype.goodie.list')
+            && auth()->can('user.goodie.edit')
+            && $goodie_enabled
+        ) {
+            $shirtSize = $goodie_tshirt
+                ? ($member->personalData->shirt_size ?? '-')
+                : '';
+            $got_goodie_button_title = $member->state->got_goodie
+                ? ($goodie_tshirt
+                    ? __('Remove T-shirt')
+                    : __('Remove goodie'))
+                : ($goodie_tshirt
+                    ? __('user.got_shirt')
+                    : __('user.got_goodie'));
+            $goodie_actions[] = ($shirtSize !== '-') ? form(
```

Comment:
Actions are not replaced by a hint on angeltype page when not set, should eventually be something like `!$shirtSize` ? (edit should of course remain)

Reply:
can't reproduce

Reply:
Happens after the user is saved via the "edit user" from profile